core/src/actions/records.ts

```diff
@@ -143,6 +143,24 @@ export class RecordCreate extends AuthenticatedAction {
     if (params.properties) {
       await record.addOrUpdateProperties(params.properties);
     }
+
+    await record.markPending();
+    await record.import();
```

Comment:
This will also catch any other errors that would be found on import, like a a secondary property not being unique.  Whatever error is thrown will be shown to the user